File: 912. Sort an Array/main.py
import heapq

class Solution:
    def sortArray(self, nums):
        # Create a min-heap
        heap = []
        
        # Push all elements into the heap
        for num in nums:
            heapq.heappush(heap, num)
        
        # Pop elements from the heap and place them back into nums
        for i in range(len(nums)):
            nums[i] = heapq.heappop(heap)
        
        return nums

File: 912. Sort an Array/main.cpp
class Solution {
public:
    vector<int> sortArray(vector<int>& nums) {
        priority_queue<int, vector<int>,greater<int>>pr;

        for(int i=0;i<nums.size();i++){
            pr.push(nums[i]);
        }

        vector<int>ans;

        int size=pr.size();
        for(int i=0;i<size;i++){
            nums[i]=pr.top();
            pr.pop();
        }
        return nums;
    }
};